### PR TITLE
Fix failing unit test in prerelease build

### PR DESCRIPTION
**Pull request by xinnanlisf**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Comment by xinnanlisf** [comment]
@MatthewBlanski, sorry for the prettier changes. The fix is in hhAccountNewChildContactNotResetPrimaryContact().

**Review by MatthewBlanski — DISMISSED** [review]

**Comment by spinkelman** [comment]
QA Passes - Will merge on Wed/Thurs after the merge freeze is lifted.

**Review by MatthewBlanski — APPROVED** [review]

**Pull request merged**
